applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml

```diff
@@ -18,8 +18,11 @@ description: |-
                 text="the relevant documentation") }}}.
 
 rationale: |-
-    <tt>aescbc</tt> is currently the strongest encryption provider, it should
-    be preferred over other providers.
+    Where etcd encryption is used, it is important to ensure that the
+    appropriate set of encrption providers is used. Currently, aescbc, kms
```

Comment:
let's fix the typo :-)

Reply:
Hah! I had actually fixed a different one already.